main.cpp

```diff
@@ -301,6 +309,22 @@ float DishWasher::printMaxDishElements()
 }
 
 
+DishWashingProcess DishWasher::checkErrors(int errorTime)
+{
+    DishWashingProcess process(false);
+    int timeMarker{0};
+    while (timeMarker < maxWashingDuration)
+    {
+        if(errorTime == timeMarker) {
+            process.errorAtMinute = errorTime;
+            return process;
+        }  
+        timeMarker += 1;
+    }
```

Comment:
curly braces go on their own line
can you add some std::cout statements inside the for() loop that print something out that shows your loop incrementing?  Maybe print out `timeMarker`
- [ ] revise